### ISSUE TITLE
Trying to set "action_json_path" results in Error

### ISSUE THREAD

**Issue by Kyniverse** [GodotVR/godot_openvr#130]
First of all, amazing plugin! It's been a blast getting this to work out of the box for the majority! I'm currently playing around a little bit with this plugin within the Mono build of Godot and didn't run into any major issues, up till now. 

I was reading through the wiki following the actions implementation to the point where it was mentioned to use the property call:
```gdscript
openVRConfig.action_json_path = "res://resources/OvrActions/actions.json"
```
Trying to call this property setter results in Godot prompting with the following error message:
```
Invalid set index 'action_json_path' (on base: 'Reference (OpenVRConfig.gdns)') with value of type 'String'.
```

After quickly scimming through the code-base here on Github I couldn't help but notice that this property is nowhere defined. Maybe I'm just missing something but as far as I can see it's currently unimplemented in the source file: 
[`OpenVRConfig.cpp`](https://github.com/GodotVR/godot_openvr/blob/master/src/open_vr/OpenVRConfig.cpp)

The result of this is currently unable to set a custom path for the defined json actions.

**Comment by beniwtv** [comment]
Yeah, seems that we never implemented setting this. Right now, the plugin only checks the default paths:
https://github.com/GodotVR/godot_openvr/blob/master/src/open_vr/openvr_data.cpp#L185

**Comment by BastiaanOlij** [comment]
I think this is actually a leftover from the original implementation, once we realized we had to export the files separately I think we just implemented it with the fixed locations:
- action folder at the location of executable
- action folder in project root
- action folder in the plugin folder

So we may need to remove that setting all together.

**Comment by BastiaanOlij** [comment]
As far as I can see this was indeed removed, are you using an older version of the plugin?

**Comment by ZYRKdev** [comment]
I was using the latest version of the plugin which I downloaded from the Releases here on Github whilst following the OpenVR Actions rundown in the Wiki. Seeing how the feature has been removed I guess the documentation/wiki part hasn't been updated in order to represent the changes?

**Comment by BastiaanOlij** [comment]
@ZYRKdev ah, yes that makes sense, that wiki was written around the time the first version of the action system was made available, before we figured out the needs of the export system.

I need to find some time to update it

**Comment by vilhalmer** [comment]
Changing the manifest path works after #156.
